cms/envs/common.py

```diff
@@ -1513,6 +1513,9 @@
     'openedx.core.djangoapps.content.learning_sequences.apps.LearningSequencesConfig',
 
     'ratelimitbackend',
+
+    # Event Tracking library, xAPI
+    'eventtracking',
```

Comment:
@feanil please verify app name should be eventtracking, not event-tracking. I'm not the most confident in my knowledge of Django to know how if finds apps.

Reply:
@jinder1s The app should be `event_routing_backends` which is the name of the module in the library with the app.  But you should reference the [app class](https://github.com/edx/event-routing-backends/blob/master/event_routing_backends/apps.py#L8) directly.  That's the new way that it's done in django.

See these docs for more info: https://docs.djangoproject.com/en/3.1/ref/applications/#configuring-applications